test/graphql/delete_note_test.rb

```diff
@@ -0,0 +1,61 @@
+require 'test_helper'
+
+class Mutations::DeleteNoteTest < ActiveSupport::TestCase
+  def perform(args = {})
+    Mutations::DeleteNote.new(object: nil, context: @context).resolve(args)
+  end
+
+  #TODO: Move this to a helper
+  def sign_in
+    user = create(:user)
+    
+    authenticated = Mutations::SignInUser.new(object: nil, context: {session: {}}).resolve(
+      email: {
+        email: user.email,
+        password: user.password
+      }
+    )
+
+    @context[:current_user] = authenticated[:user]
+  end
+
+  test 'failure w/no credentials' do
+    assert_raises(GraphQL::ExecutionError) do
+      Mutations::DeleteNote.new(object: nil, context: {}).ready?
+    end
+  end
+
+  test 'successfully removes recipe note' do
+    sign_in
+
+    note = create(:recipe_note)
+    recipe = note.note_subject
+
+    assert_difference('Note.count', -1) do
```

Comment:
you can oneline this stuff if you prefer less vertical scrolling (you don't use `result`):

```suggestion
    assert_difference('Note.count', -1) { perform(id: note.id) }
```